daemon/container.go

```diff
@@ -291,6 +291,22 @@ func (daemon *Daemon) verifyContainerSettings(platform string, hostConfig *conta
 		return nil, nil
 	}
 
+	if hostConfig.Privileged {
+		for _, deviceMapping := range hostConfig.Devices {
+			if deviceMapping.PathOnHost == deviceMapping.PathInContainer {
+				continue
+			}
+			if _, err := os.Stat(deviceMapping.PathInContainer); os.IsNotExist(err) {
+				continue
+			} else {
```

Comment:
This might be cleaner like:

```go
if _, err := os.Stat(...); err != nil {
  if os.IsNotExist(err) {
    continue
  }
  return nil, errors.Wrap(err, "error stating device path in container")
}
return nil, errors.Errorf(...)
```

Reply:
Looks better, tks